### PR TITLE
Register Google Gemini Models

### PR DESCRIPTION
**Pull request by apandy02**
This change uses Google's OpenAI compatible API endpoint and adds google gemini models to the ell framework.

### CODE REVIEW

File: src/ell/models/google.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    Note:
+        The function doesn't return anything but updates the global
+        configuration with the registered models.
+    """
+    standard_models = [
+    'gemini-1.5-flash',
+    'gemini-1.5-flash-8b',
+    'gemini-1.5-pro',
+    'gemini-1.0-pro',
```
Comment (aryamanpandya99) [inline src/ell/models/google.py:52]: should we include experimental models here?

**Comment by vedkm** [comment]
Does this work with tools?

**Comment by apandy02** [comment]
@vedkm not at this time, I believe for this the provider would have to be implemented

**Comment by MadcowD** [comment]
This looks great for now!

**Pull request merged**